presto-main/src/main/java/com/facebook/presto/operator/ExchangeClientConfig.java

```diff
@@ -91,6 +93,32 @@ public ExchangeClientConfig setMaxErrorDuration(Duration maxErrorDuration)
         return this;
     }
 
+    public boolean isAsyncPageTransportEnabled()
```

Comment:
nit: please reorder the getters setter in the same order as the properties are declared.